Makefile

```diff
@@ -858,7 +858,9 @@ install.completions:
 .PHONY: install.docker
 install.docker:
 	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
-	install ${SELINUXOPT} -m 755 docker $(DESTDIR)$(BINDIR)/docker
+	$(eval COMPILED_DOCKER_SCRIPT := $(shell mktemp))
```

Comment:
The word `COMPILED` is misleading here. Can you think of another word that might be clearer? (This is not a blocker)